### DISCUSSION TITLE
Compiler Issue of C

### DISCUSSION THREAD

**Discussion by aditii144** [aditii144/c-#5] [Q&A] [answered]
Is there any trusted compiler for c programming? 
I use Vs code and that one compiler I downloaded from Sourceforge website sucks!!!

**Comment by ujjwalbhandarii** [comment, marked as answer]
I use one from [https://winlibs.com/](winLibs) and it works really well. 

you can download any of these. 
![image](https://user-images.githubusercontent.com/80826873/197403787-1768e03a-3678-48eb-a99f-f7f1893e57b4.png)
 
I have this one: GCC 12.2.0 + LLVM/Clang/LLD/LLDB 14.0.6 + MinGW-w64 10.0.0 (UCRT) - release 2  

and after downloading you need to extract its file and after that setup environment variables. Then you are good to go.
